subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv.c

```diff
@@ -1003,8 +1003,10 @@ static void isr_tx(void *param)
 #if defined(CONFIG_BT_CTLR_ADV_EXT)
 	struct lll_adv *lll = param;
 	uint8_t phy_p = lll->phy_p;
+	uint8_t phy_flags = lll->phy_flags;
 #else
-	uint8_t phy_p = 0;
+	uint8_t phy_p = 0U;
+	uint8_t phy_flags = 0U;
```

Comment:
More of a question: Would declaring these as `const` provide any performance gains, or any other reason why to declare these as `const`?